semantic_release/ci_checks.py

```diff
@@ -106,3 +118,5 @@ def check(branch: str = 'master'):
         circle(branch)
     elif os.environ.get('GITLAB_CI') == 'true':
         gitlab(branch)
+    elif os.environ.get('CI') == 'true':
```

Comment:
This should use something specific for bitbucket since most services set `CI=true`.

Reply:
I see, there is no 'BITBUCKET_CI' though, it could be another kind of condition like: `'BITBUCKET_BUILD_NUMBER'  in os.environ` ? https://confluence.atlassian.com/bitbucket/environment-variables-in-bitbucket-pipelines-794502608.html

Reply:
Sounds like a good idea 👍

Reply:
Done